export.py

```diff
@@ -82,17 +87,54 @@ def json_to_markdown(json):
     json['body'] = TAG.sub('', body).strip()
 
     json['slug'] = get_slug(json)
-    json['subject'] = json['subject'] or json['date']
+    json['subject'] = json['subject'] or json['eventtime']
 
     return """id: {id}
 title: {subject}
 slug: {slug}
-date: {date}{tags}
+date: {eventtime}{tags}
 
 {body}
 """.format(**json)
 
 
+def json_to_alt_markdown(json):
+    body = TAGLESS_NEWLINES.sub('<br>', json['body'])
+
+    h = html2text.HTML2Text()
+    h.body_width = 0
+    h.unicode_snob = True
+    body = h.handle(body)
+    body = NEWLINES.sub('\n\n', body)
+
+    # read UTX tags
+    tags = TAG.findall(body)
+    json['tags'] = len(tags) and '\ntags: {0}'.format(', '.join(tags)) or ''
+
+    # remove UTX tags from text
+    json['body'] = TAG.sub('', body).strip()
+
+    json['slug'] = get_slug(json)
+    json['subject'] = json['subject'] or json['eventtime']
+
+    # get post time
+    post_date = str(datetime.strptime(json['eventtime'],"%Y-%m-%d %H:%M:%S"))
+    post_time = post_date[11:16] # Post's time = HH:MM
```

Comment:
Rather than cutting sections of the string and putting the explanations in the comments, it would be much cleaner to get the exact string right away:
```python
post_time = str(datetime.strptime(json['eventtime'],"%H:%M"))
```
The same applies to a few other places below in the code.